index.js

```diff
@@ -58,6 +60,30 @@ app.post("/github_push_webhook/:id", async (req, res) => {
   }
 });
 
+app.post("/github_discord_urls", async (req, res) => {
+  try {
+    const reqBody = req.body;
+    const data = await pool.query(
+      `insert into github_discord_url (discord_url) values ($1)`,
+      [reqBody.url]
+    );
+
+    const id = await pool.query(
```

Comment:
use existing function

Reply:
Can't, The old function is getting "discord_url" from "id" but in this case I need "id" from "discord_url"

Reply:
Making a new function to get_id...discord_url